src/Controls/src/Build.Tasks/ErrorMessages.resx

```diff
@@ -243,6 +243,11 @@
   <data name="XDataTypeSyntax" xml:space="preserve">
     <value>x:DataType expects a string literal, an {{x:Type}} markup or {{x:Nul}l}.</value>
   </data>
+  <data name="UnattributedMarkupType" xml:space="preserve">
+    <value>Consider attributing the markup extension "{0}" with [RequireService] or [AcceptEmptyServiceProvider] if it doesn't require any.</value>
```

Comment:
I created ticket because I think our localization is broken and these aren't getting translated, just a note I wanted to had so we can keep track of it. 

https://github.com/dotnet/maui/issues/20515